geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/RasterRDD.scala

```diff
@@ -129,8 +167,48 @@ class ProjectedRasterRDD(val rdd: RDD[(ProjectedExtent, MultibandTile)]) extends
     new ProjectedRasterRDD(rdd.reproject(crs, resample))
   }
 
+  def reclassify(
+    breakMap: BreakMap[Int, Int]
+  ): RasterRDD[ProjectedExtent] = {
+    val reclassifiedRDD = {
+      rdd.map {
+        case (k: ProjectedExtent, m: MultibandTile) =>
+        val count = m.bandCount
+        val tiles = Array.ofDim[Tile](count)
+
+        for (x <- 0 until count) {
+          val band = m.band(x)
+          tiles(x) = band.map(i => breakMap.apply(i))
+        }
+
+        (k, MultibandTile(tiles))
+      }
+    }
+
+    ProjectedRasterRDD(reclassifiedRDD)
```

Comment:
Can this be implemented in the abstract class? We don't have to do anything to the key, so we should be able to just pass it through.